### DISCUSSION TITLE
Title and description

### DISCUSSION THREAD

**Discussion by expressmoney** [viewflow/viewflow#416] [General]
It will be correct if the title property is displayed in the Task column and the description property in the Brief column. 
![viewflow title](https://github.com/viewflow/viewflow/assets/139622411/69c5fffa-9bf2-40a5-8a96-86b8c785bfa8)

**Comment by kmmbvnr** [comment]
Indeed, the title of the task is currently not being utilized in the list views, and this issue needs to be addressed.

However, it's unclear what occurs in your case with the task_description.

If the task description is present, it is assigned to the brief as seen in this link to the Viewflow workflow models on GitHub:

**Comment by kmmbvnr** [comment]
Improved it d9f8cc4
